### ISSUE TITLE
How to visualize loss curve

### ISSUE THREAD

**Issue by zxin8218** [eriklindernoren/PyTorch-YOLOv3#469]
I have finished training, but how to visualize the loss curve to see if it converges

**Comment by www7890** [comment]
Tensorboard is included in this repo. You may want to take a look.

**Comment by xiaoyuzhu666** [comment]
您好，请问test loss怎么计算？代码里好像只有train loss，没有test loss。。

**Comment by zxin8218** [comment]
> p

It's ok! Thank you

**Comment by zxin8218** [comment]
> Tensorboard is included in this repo. You may want to take a look.

It's ok! Thank you
